sql-plugin/src/main/scala/com/nvidia/spark/rapids/CostBasedOptimizer.scala

```diff
@@ -279,21 +308,23 @@ class DefaultCostModel(conf: RapidsConf) extends CostModel {
           // different CAST operations have different costs, so we allow these to be configured
           // based on the data types involved
           expr.conf.getExpressionCost(s"Cast${cast.fromType}To${cast.toType}")
-              .getOrElse(conf.defaultExpressionCost)
+            .getOrElse(conf.defaultExpressionCost)
         case _ =>
           // many of our BaseExprMeta implementations are anonymous classes so we look directly at
           // the wrapped expressions in some cases
           expr.wrapped match {
-            case _: AttributeReference => 1.0 // no benefit on GPU
-            case Alias(_: AttributeReference, _) => 1.0 // no benefit on GPU
+            case _: AttributeReference => 0.01 // no benefit on GPU
```

Comment:
Actually the cost of an AttributeReference here is probably close to free, because we are just incrementing a reference count, but the cost of a GetStructField needs to copy the data out, so they should be different.